Manifests/ManagedPreferencesApplications/net.glencode.Particulars.plist

```diff
@@ -488,6 +560,38 @@
 			<key>pfm_type</key>
 			<string>boolean</string>
 		</dict>
+		<dict>
+			<key>pfm_app_min</key>
+			<string>58</string>
+			<key>pfm_default</key>
+			<false/>
+			<key>pfm_description</key>
+			<string>Displays an error message if no active network interfaces are found. (Assuming network is enabled in detailLevel.)</string>
```

Comment:
Everything looks good. My only issue is with the punctuation in the description here - let's change it to the following?

Displays an error message if no active network interfaces are found (assuming network is enabled in `detailLevel`).

Reply:
Requested changes have been made